src/status_im/ui/screens/home/views.cljs

```diff
@@ -113,22 +114,24 @@
             #(re-frame/dispatch [:init-rest-of-chats])
             100))))}
     [react/view styles/container
-     [toolbar show-welcome? (and network-initialized? (not rpc-network?)) sync-state latest-block-number (not logging-in?)]
-     (cond show-welcome?
-           [welcome view-id]
-           loading?
-           [react/view {:style {:flex            1
-                                :justify-content :center
-                                :align-items     :center}}
-            [connectivity/connectivity-view]
-            [components/activity-indicator {:flex 1
-                                            :animating true}]]
-           :else
-           [react/view {:style {:flex 1}}
-            [connectivity/connectivity-view]
-            [chats-list]])
-     (when platform/android?
-       [home-action-button (not logging-in?)])]))
+     [status-bar/status-bar {:type :main}]
+     [react/view (assoc styles/container :background-color :white)
```

Comment:
I suppose you'll have to be careful when rebasing the background-color PR with this one

Reply:
yes sure, but if I don't add background here it will be gray... Also i'm not sure when exactly background PR will be merged, animations look off in stack navigators because bg is technically transparent. So there is some work to be done, it might take some time